packaging/dpkg/debian/control

```diff
@@ -21,8 +21,8 @@ Build-Depends: debhelper,
                openssl,
                php-all-dev,
                php-cli,
-               python3 (>= 3.12),
-               python3-dev (>= 3.12),
+               python3 (>= 3.12) <!no-python312>,
```

Comment:
This allow to build with `no-python312` build profile, without depending on python3.12+